### DISCUSSION TITLE
support for Windows 11

### DISCUSSION THREAD

**Discussion by alfredosuriano** [File-New-Project/EarTrumpet#1033] [Ideas]
I use EarTrumpet since two years and it's amazing! 
I have just installed Windows 11 and i can't hide Volume icons from taskbar (as i did in win10), so it displays two sound icons.
Can you please improve compatibility with Windows 11 to fix this problem?
Thanks a lot!
![image](https://user-images.githubusercontent.com/98228835/150649109-9e017281-d2a5-4494-a5e6-910505125884.png)

**Comment by riverar** [comment]
Hi @asuSlash, this is not a compatibility issue. Windows 11 does not let you turn off or remove these icons sadly.

**Comment by lichtmetzger** [comment]
You can run ExplorerPatcher though to switch back to the Windows 10 style taskbar.

With ExplorerPatcher active, run Win+R and enter this command:
shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}

This will open the old dialog where you can turn off system symbols. Disable the volume icon here - tadaa, now you only have EarTrumpet in your taskbar. :)
